src/components/asynccontent/AsyncContent.js

```diff
@@ -2,12 +2,40 @@ import React, { PropTypes } from 'react';
 import FontAwesome from 'react-fontawesome';
 import { Alert } from 'react-bootstrap';
 
+export const ASYNC_STATUS = Object.freeze({
```

Comment:
It's the simplest way to get an enumeration in Javascript. `Symbol('foo') !== Symbol('foo')`, which prevents accidental collisions that can occur with strings. `Object.freeze()` makes `ASYNC_STATUS` immutable